pyiron_base/database/generic.py

```diff
@@ -38,6 +41,121 @@
 __status__ = "production"
 __date__ = "Sep 1, 2017"
 
+class IsDatabase(ABC):
+    """
+    Captures common interface for all database types in pyiron, e.g. SQL/SQLite/FileTable.
+    """
+
+    @abstractmethod
+    def _get_view_mode(self):
+        pass
+
+    @property
+    def view_mode(self):
+        """
+        Get viewer_mode - if viewer_moded is enable pyiron has read only access to the database.
```

Comment:
Docstring still uses deprecated method name

Reply:
I fixed this one, but there's a similar inconsistency on `Project` with `switch_to_viewer_mode`.  I left that for now as it's externally visible.